app/javascript/lib/tagging/index.js

```diff
@@ -61,6 +61,14 @@ export function initMultiSelectVitaPartner() {
             },
         }
     });
+
+    input.addEventListener("change", (event) => {
+        let { value } = event.target
+        if (value) {
+            value = JSON.stringify(JSON.parse(event.target.value).map(v => v.id))
+        }
+        document.querySelector("#vita_partners").value = value
```

Comment:
Sorry if I'm not understanding this correctly, but will this overwrite the current value of the #vita_partners hidden field, rather than append to it?

Reply:
Nevermind - it looks like because this is a multiselect, the value includes all currently selected partners. Is that right?

Reply:
Yes - the value in the input is overwritten and includes all currently selected partners.